src/inmanta/ast/statements/call.py

```diff
@@ -238,6 +238,11 @@ def call_in_context(
             except UnknownException as e:
                 result.set_value(e.unknown, self.ast_node.location)
             except UnsetException as e:
+                call: str = "{module}::{func}".format(module=self.plugin.__module__, func=self.plugin.__function_name__)
```

Comment:
I would use an f-string here (and in the next line)

```suggestion 
                module = self.plugin.__module__
                func = self.plugin.__function_name__
                call: str = f"{module}::{func}"
```